pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go

```diff
@@ -539,6 +540,76 @@ func TestWithBinding(t *testing.T) {
 	}
 }
 
+func TestIsSchedulableAfterStorageClassAdded(t *testing.T) {
+	var modeWait = storagev1.VolumeBindingWaitForFirstConsumer
+	pvLister := tf.PersistentVolumeLister{
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "Vol_1", Labels: map[string]string{v1.LabelFailureDomainBetaZone: "us-west1-a"}},
+		},
+	}
+
+	pvcLister := tf.PersistentVolumeClaimLister{
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "PVC_1", Namespace: "default"},
+			Spec:       v1.PersistentVolumeClaimSpec{VolumeName: "Vol_1"},
+		},
+	}
+
+	scLister := tf.StorageClassLister{
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "SC_1", Namespace: "default"},
+		},
+	}
```

Comment:
Do we need them?

Reply:
We don't use it, so it can be removed. I removed lister part.